puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: {timeout}
```

Comment:
I'm not sure what's happening here precisely, perhaps a lack of quoting?  In the XML output I see:

    &lt;timeoutMinutes>OrderedDict([('timeout', None)])</timeoutMinutes>